### ISSUE TITLE
Wrong activation in discriminator

### ISSUE THREAD

**Issue by woitee** [hcnoh/WGAN-tensorflow2#1]
Discriminator in WGAN should have linear output activation, not sigmoid.
(src: https://wiseodd.github.io/techblog/2017/02/04/wasserstein-gan/ or https://machinelearningmastery.com/how-to-implement-wasserstein-loss-for-generative-adversarial-networks/, a little more complex to find in the original paper)

It (surprisingly?) still works after I change it, though :).

**Comment by hcnoh** [comment]
Thank you for your comment. I will check those materials and modify the output activation.

Additionally, would you mind if I ask you to show a generation result after changing the output activation?

**Comment by woitee** [comment]
I have refactored the code so much beyond (I struggled with making WGAN work myself, so I got this code to help me find my error), and I didn't keep any of the images, so it would take time to get the generation result with just the one modification...

Sorry for not helping that way, but I feel it will take you exactly the same time it would take me.

**Comment by woitee** [comment]
what I have right now, is 5 random images after 20 epochs of training

![image](https://user-images.githubusercontent.com/4940039/101277836-7ff15f80-37b7-11eb-9437-70e1925806f6.png)

**Comment by hcnoh** [comment]
Oh, I think your result is better than mine. I am beginning to look forward to your generation result. 👍 

I will check the paper again and tell you the exact activation in discriminator.

Thank you.
